packaging/crio.conf.d/microshift.conf

```diff
@@ -5,10 +5,6 @@ conmon_cgroup = "pod"
 cgroup_manager = "systemd"
 
 [crio.network]
-# cbr0 is the name configured by flannel in /etc/cni/net.d/ config file
-# by declaring this crio will wait until that network is configured.
-cni_default_network = "cbr0"
```

Comment:
Do we need to pick a different default, or is that configured some other way?

Reply:
We faced an issue with pods starting being attached to two different bridges (CRI-O bridge and Flannel bridge) due to a race condition. We decided to fix it by letting CRI-O wait for the Flannel bridge to be created and attach all pods there. Now that Flannel is gone, we can just rely on the default bridge from CRI-O. 

Short answer, I don't think we need to add any default there.

Reply:
Thanks, that makes sense.